struts2-archetype-portlet/pom.xml

```diff
@@ -19,7 +19,7 @@
         <resources>
             <resource>
                 <directory>src/main/resources</directory>
-                <filtering>false</filtering>
+                <filtering>true</filtering>
```

Comment:
I think this was done on purpose to avoid processing archetypes' templates which resides in the folder - with `filtering` set to `true` all the `${...}` will be translated during building the archetypes, but the case is that this must happen when a user is using them to generate a new project.

Reply:
The portlet archetype appears to need the `true` value for the resulting project to work, though.  With `false` the pom.xml that gets built references non-existent variable names (and the "resulting project" created by the archetype fails to build as a result).

The above change to `true` agrees with/matches the change in your current PR (https://github.com/apache/struts-archetypes/blob/db3da1e3032a1605c2b35d88bedc749c1a6228d0/struts2-archetype-plugin/pom.xml), as it is set to `true` there as well.

I think the change is the same/correct in both PRs correct and it should be `true`.  Does that make sense ? 😕

Reply:
Ach... my mistake, I thought it will affect all the archetypes. And it's a good question why I used `true` there 🤔